apps/files/src/components/FileList.vue

```diff
@@ -210,11 +217,16 @@ export default {
       }
 
       return null
+    },
+
+    $_disabledActionSpinnerTooltip (item) {
+      const message = this.$gettext('There is currently an action in progress for this %{itemType}')
+      return this.$gettextInterpolate(message, { itemType: item.type })
```

Comment:
item.type is either file or folder - this shall be translated as well.

Reply:
Right - fixed 👍